server/public/styles.css

```diff
@@ -0,0 +1,31 @@
+#header {
+  color: white;
+}
+
+.card-img-top {
+  object-fit: scale-down;
+}
+
+.card-container {
+  height: 350px;
+}
+
+.card-img-container {
+  height: 200px;
+}
+
+.card-title {
+  font-size: 14px;
+}
+
+.card-body {
+  padding: 0.6rem;
+  line-height: 1rem;
+}
+.card-text {
+  font-size: 10px;
+}
+
+.card {
+  padding: 0;
+}
```

Comment:
Don't modify Bootstrap's classes, instead make custom ones. Even better some of these styles are supported by their utility classes.

https://getbootstrap.com/docs/4.4/utilities/spacing/